src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,26 +40,76 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean =
+      if (b) false
+      else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean =

```

Comment:
Could be done much simpler: two cases instead of three.
Also bear in mind that `and` should behave as Java's `&&` means if `left` is false then `right` is not calculated.